README.md

```diff
@@ -8,7 +8,7 @@
 
```

Comment:
**[Optional]**
- [ ] Your summary is descriptive but it would nice if you add some more details in it as you can see in the image below :arrow_down: to show your professionalism with your work :+1:

![image](https://user-images.githubusercontent.com/93039359/194324887-4b36055c-9b2b-47f4-ac0e-19bc39574c64.png)